test/xpu_api/utilities/pairs/pairs.pair/default_constructible_g.pass.cpp

```diff
@@ -0,0 +1,59 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/type_traits>
+#include <oneapi/dpl/utility>
+
+#include "support/test_macros.h"
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+bool
+test_default_constructible()
```

Comment:
I guess the purpose of the test is the opposite:)
```suggestion
test_non_default_constructible()
```